src/views/Farms/Farms.tsx

```diff
@@ -52,9 +67,19 @@ const Farms: React.FC = () => {
 
         return { ...farm, apy }
       })
-      return farmsToDisplayWithAPY.map((farm) => <FarmCard key={farm.pid} farm={farm} removed={removed} />)
+      return farmsToDisplayWithAPY.map((farm) => (
+        <FarmCard
+          key={farm.pid}
+          farm={farm}
+          removed={removed}
+          bnbPrice={bnbPrice}
+          cakePrice={cakePrice}
+          ethereum={ethereum}
+          account={account}
+        />
+      ))
     },
-    [bnbPrice, farmsLP],
+    [bnbPrice, farmsLP, account, cakePrice, ethereum],
```

Comment:
This function is pretty damn nasty to start with, I just added dependencies to the hook as they're required by the new farm cards

Reply:
👍